packages/marko/src/core-tags/migrate/util/import-tag.js

```diff
@@ -1,25 +1,38 @@
 const camelCase = require("camelcase");
 const nameReg = /\/([^/]+?)(?:\/index)?(?:\.[^/]+)?$/;
 
-module.exports = function importTag(baseImportPath, context) {
+module.exports = function importTag(importIdentifier, context) {
   const builder = context.builder;
   const migrateImports = (context._migrateImports = context._migrateImports || {
     identifiers: {},
     paths: {}
   });
-  const importPath = context.getRelativePath(baseImportPath);
-
-  if (migrateImports.paths[importPath]) {
-    return migrateImports.paths[importPath];
-  }
-
-  const match = nameReg.exec(importPath);
+  let isImportedAsTag = importIdentifier[0] === "<";
```

Comment:
Minor nitpick, but we should probably make this a `const` to be consistent.

Reply:
ah yes good catch